HelpSource/Tutorials/QT/things_you_wanna_know.schelp

```diff
@@ -0,0 +1,61 @@
+title:: QT : Things you probably want to know
+summary:: Miscellaneous things you probably want to know about QT.
+categories:: Tutorials>QT
+
+section:: Rendering isn't always available
+
+SuperCollider provides some useful classes to automate processes. Some of them are associated to a link::Classes/Clock:: : mainly link::Classes/Routine:: and link::Classes/Task::.
+
+strong::QT rendering system is unavailable within the context of clock associated threads::. In other words, you cannot 'directly' modify a widget within a link::Classes/Routine:: or a link::Classes/Task::.
+
+Instead, you need to defer the modification by strong::encapsulating it in a deferred function::, that will be executed once SuperCollider resume its main context. Doing otherwise will crash the program :
+
+code::
+(
+var win = Window();
+var slider= Slider();
+var value = 0;
+
+var routine = Routine({
+	loop {
+		value = 101.rand / 100;
+		// Interacting with a widget from within a routine :
+		// instructions must be nested in a function, and deferred
+		{ slider.value_(value); }.defer;
+		0.5.wait;
+	};
+}).play;
+
+win.layout_(
+	VLayout()
+	.add(slider)
+);
+
+CmdPeriod.doOnce({
+	routine.stop;
+	win.close;
+});
+
+win.front;
+)
+::
+
+section:: QT limitations
+
+This remark mostly concerns debugging and issues understanding. Since strong::QT is a third-party program integrated within SuperCollider::, some of its limitations are unrelated to SuperCollider itself.
```

Comment:
```suggestion
This remark mostly concerns debugging and understanding issues. Since strong::Qt is a third-party library integrated within SuperCollider::, some of its limitations are unrelated to SuperCollider itself.
```